interface.go

```diff
@@ -16,33 +17,49 @@ type TableInfo struct {
 }
 
 type ColumnDef struct {
-	Path string    `json:"path"`
+	Name string    `json:"name"`
+	Key  string    `json:"key"`
 	Type FieldType `json:"type"`
 }
 
 type TableDriver interface {
 	New(name string, columns []ColumnDef) (TableStore, error)
 	Get(name string) (TableStore, error)
+	GetAllColNames() chan string
+	GetLabels(edges bool) chan string
 	List() []string
+	Delete(name string) error
 	Close()
 }
 
-type Entry struct {
-	Key   []byte
-	Value map[string]any
+type Row struct {
+	Id    []byte
+	Label string
```

Comment:
Is this label? It seems like it's the table name in the bulk load

Reply:
Yep table name and label are the same thing in benchtop. It's used in https://github.com/bmeg/benchtop/blob/feature/refactor-kvindex/bsontable/driver.go#L345-L361 to create the tables needed to load the graph elements.